plasma_framework/migrations/1_initial_checks_and_migrations.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+/* eslint-disable prefer-destructuring */
+
+const Migrations = artifacts.require('Migrations');
+
+module.exports = async (deployer, network, accounts) => {
+    // Addresses
+    global.deployerAddress = accounts[0];
```

Comment:
we actually have three address here, one more `maintainer`. See this: https://github.com/omisego/plasma-contracts/issues/233

aside from `initAuthority()`, all other functions currently using `authorityAddress` should be done with `maintainer address` instead